packages/web/components/RegisterPlayer.tsx

```diff
@@ -0,0 +1,34 @@
+import {
+  Button,
+  ButtonGroup,
+  EmailIcon,
+  Icon3box,
+  MetaHeading,
+} from '@metafam/ds';
+import React from 'react';
+
+import { FlexContainer } from './Container';
+
+export const RegisterPlayer: React.FC = () => {
+  return (
+    <FlexContainer flex={1}>
+      <MetaHeading m={5}>Register your Player account</MetaHeading>
+      <ButtonGroup spacing={5} mt={20}>
+        <RegisterButton textTransform="uppercase">
+          <Icon3box mr={2} /> box
```

Comment:
As mentioned above, they have changed how icons work in chakra-ui@next. 
And when I use them as `leftIcon`, the padding they had set doesn't work and the icons are not center aligned horizontally. This can be seen by running story-book where icons are used as `leftIcon` and `rightIcon`.
Ref: https://next.chakra-ui.com/docs/migration#button & https://github.com/chakra-ui/chakra-ui/issues/1024
Hence, I've used the icons as children with margin set on them.